scripts/tests/tests.sh

```diff
@@ -0,0 +1,25 @@
+# SANITY CHECK tests
+# This is a temporary solution we want something more standard then a custom script 
+# in the future.
+# This tests could take a large amount of time
+
+VERSION=0.23.0
+
+# Generate modules
+python3 ../generate_modules.py $VERSION --output modules/spack
+module use modules
+module load spack/$VERSION
+
+# Config sanity checks
+echo "spack version found: $(spack --version | grep $VERSION | wc -l)"
+echo "gcc 11 compiler found: $(spack compilers | grep gcc@11 | wc -l)"
+echo "gcc 10 compiler found: $(spack compilers | grep gcc@11 | wc -l)"
+echo "cce 15 compilers found: $(spack compilers | grep cce@15 | wc -l)"
```

Comment:
I'm not sure I understand what this output is meant to convey -- is it to verify that one version of each is found? The GCC 10 line is also counting the number of occurrences of `gcc@11`.

Reply:
Nice spot. This is a basic check that the compilers.yaml was picked up. So it checks that the expected compilers are there. They should all report one.